core/src/ast/function.rs

```diff
@@ -6,6 +6,8 @@ use {
 
 #[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize, Display)]
 pub enum Function {
+    #[strum(to_string = "ABS")]
+    ABS(Expr),
```

Comment:
Let's simply follow the Rust's naming patterns.
`Abs` would be good to use for this.

Reply:
Thanks..  will make necessary modifications.